### PR TITLE
Restrictions for roles

### PR DESCRIPTION
**Pull request by Levia**
This is about restricting redundant roles selection.
If `Super Delegate` is selected, we don't want `Delegate` to be selected as well; same thing applies for Admin and Respondent Admin.

I've also adjusted the roles order and look on the new/edit user page

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -24,6 +24,14 @@
                           puts "Created respondent admin role"
                         end
 
+Role.find_by_name('admin').update_attributes(order_index: 1)
```
Comment (spencerldixon) [inline db/seeds.rb]: I would move these inside the `first_or_create` block so they're added when the role gets created. Maybe do a migration for updating the current ones

Comment (spencerldixon) [reply db/seeds.rb]: Other than that, all good to merge!

Comment (Levia) [reply db/seeds.rb]: Thanks! Fixed.

File: db/seeds.rb [hunk 2]
```diff
@@ -4,26 +4,33 @@
 
 # Create roles if they don't exist
 
-admin               = Role.where(name: "admin").first_or_create do |u|
+admin               = Role.where(name: "admin").first_or_create do |r|
+                        r.update_attributes(:order_index: 1)
```
Comment (spencerldixon) [inline db/seeds.rb]: Ah you just need to do `u.order_index = 1` like in the user one

Comment (Levia) [reply db/seeds.rb]: ah sure, sorry! Fixed.

**Pull request merged**